com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/LuaBuilder.java

```diff
@@ -72,7 +72,7 @@ public abstract class LuaBuilder extends Builder {
     private static List<ILuaPreprocessor> luaPreprocessors = null;
     private static List<ILuaObfuscator> luaObfuscators = null;
 
-    private Map<String, LuaScanner> luaScanners = new HashMap();
```

Comment:
Each file has it's own builder. So one file has only one Lua Scanner. There is no reason to have more than on LuaScanner